injectScript.js

```diff
@@ -0,0 +1,60 @@
+function isEvolvLoaded() {
+    return window.evolv !== undefined &&
+        window.evolv.context !== undefined &&
+        window.evolv.context.remoteContext !== undefined &&
+        window.evolv.context.remoteContext.experiments !== undefined &&
+        window.evolv.context.remoteContext.experiments.allocations !== undefined
+}
+
+var pollingSafetyNet = 0;
+function poll() {
+    if (pollingSafetyNet++ < 50) {
+        if (!isEvolvLoaded()) return setTimeout(poll, 100);
+
+        window.postMessage({
```

Comment:
So here is where I would switch to the listener approach
on context.changed
https://media.evolv.ai/javascript-sdk/releases/latest/docs/EvolvClient.html#on

send the remote context to the extension